src/main/java/seedu/parser/Parser.java

```diff
@@ -106,7 +106,8 @@ public Command parseCommand(String userInput) {
             if (commandAndArgument.get(1) == null) {
                 return new ListCommand();
             } else {
-                return new ListCommand(new ArrayList<>(Collections.singleton(commandAndArgument.get(1))));
+                args = new ArrayList<>(Collections.singleton(commandAndArgument.get(1).toUpperCase(Locale.ROOT)));
```

Comment:
Yes, I think this should solve the problem of being unable to specify EquipmentType in lowercase.